spec/requests/items_spec.rb

```diff
@@ -84,4 +106,28 @@
       end
     end
   end
+
+  describe 'POST /items' do
+    context 'when item post request attributes are valid' do
+      valid_attributes = {name: 'Asparagus', sell_in: 10, quality: 25}
+      before { post '/api/v1/items', params: valid_attributes }
+
+      it 'returns status code 201' do
+        expect(response).to have_http_status(201)
+      end
+    end
+
+    context 'when an invalid request' do
+      valid_attributes = {title: 'Lorem Ipsum'}
+      before { post '/api/v1/items', params: valid_attributes }
+
+      it 'returns status code 422' do
+        expect(response).to have_http_status(422)
```

Comment:
Question: Why status code `422`? Is that the norm for Rails?

Reply:
Yes, rails convention. 
:unprocessable_entity | 422 Unprocessable entity. The request was formatted correctly but cannot be processed in its current form. _Often used when the specified parameters fail validation errors_  
-- | --

source: https://kapeli.com/cheat_sheets/HTTP_Status_Codes_Rails.docset/Contents/Resources/Documents/index